### PR TITLE
enh(EventView) :Improve the button consistency  with verbs in french translation on the Event view

### PR DESCRIPTION
**Pull request by CoralieRubin**
## Description
This Pr standardize buttons in the Event view.
Replacing french expression in verbs.

Before : 

Capture d’écran de 2020-07-01 11-27-44

Capture d’écran de 2020-07-01 11-28-42

Now : 

Capture d’écran de 2020-07-01 15-00-10

Capture d’écran de 2020-07-01 15-00-27

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.04.x
- [ ] 19.10.x
- [ ] 20.04.x
- [x] 20.10.x (master)

<h2> How this pull request can be tested ? </h2>

For testing this Pr you need to switch language to french and go to the event view.
The button must have a verbs action and when you selected a ressource the button pop up must be changed to.

## Checklist

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -15252,6 +15255,9 @@ msgstr "Changer l'heure de début"
 msgid "Host group"
 msgstr "Groupe d'hôtes"
 
+msgid "DownTime"
+msgstr "Planifier une maintenance"
+
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15260]:
```suggestion
```

**Comment by bdauria** [comment]
@CoralieRubin one detail: the title of the Downtime popup in french is "Temps d'arrêt", whereas the action is now called "Planifier une maintenance". I think we should rename the title to stay consistent.

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -2625,7 +2625,7 @@ msgstr "Statistiques du moteur"
 
 #: centreon-web/www/install/smarty_translate.php:942
 msgid "No downtime scheduled"
-msgstr "Pas de temps d'arrêt programmé"
+msgstr "Pas de plage de maintenance programmé"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:2628]:
```suggestion
msgstr "Pas de plage de maintenance programmée"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -5497,7 +5497,7 @@ msgstr "Options d'acquittement par défaut"
 
 #: centreon-web/www/include/Administration/parameters/engine/form.php:210
 msgid "Default downtime settings"
-msgstr "Options de temps d'arrêt par défaut"
+msgstr "Options de la maintenance par défaut"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:5500]:
```suggestion
msgstr "Options de la plage de maintenance par défaut"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -9966,7 +9966,7 @@ msgstr "Bagotant"
 #: centreon-web/www/include/configuration/configObject/contact_template_model/formContactTemplateModel.php:260
 #: centreon-web/www/include/configuration/configObject/host_template_model/formHostTemplateModel.php:630
 msgid "Downtime Scheduled"
-msgstr "Temps d'arrêt programmés"
+msgstr "Maintenances programmées"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:9969]:
```suggestion
msgstr "Plages de maintenance programmées"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -11917,7 +11917,7 @@ msgstr "Secondes"
 #: centreon-web/www/include/configuration/configDowntime/ajaxForms.php:71
 #: centreon-web/www/include/monitoring/downtime/AddDowntime.php:131
 msgid "Downtime type"
-msgstr "Type de temps d'arrêt"
+msgstr "Type de maintenance"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:11920]:
```suggestion
msgstr "Type de plage de maintenance"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -13696,7 +13696,7 @@ msgstr "Configurer un temps d'arrêt pour les services des hôtes"
 #~ msgstr "Ajouter un temps d'arrêt pour un hôte"
 
 #~ msgid "Select a downtime type"
-#~ msgstr "Sélectionner un type de temps d'arrêt"
+#~ msgstr "Sélectionner un type de maintenance"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:13699]:
```suggestion
#~ msgstr "Sélectionner un type de plage de maintenance"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -14577,10 +14577,10 @@ msgstr "Poubelle"
 #~ msgstr "Temps d'arrêt des hôtes"
 
 #~ msgid "No downtime scheduled for hosts"
-#~ msgstr "Aucun temps d'arrêt planifié pour les hôtes"
+#~ msgstr "Aucune plage de maintenance planifiée pour les hôtes"
 
 #~ msgid "View downtimes of services"
-#~ msgstr "Visualiser les temps d'arrêt des services"
+#~ msgstr "Visualiser les maintenances des services"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14583]:
```suggestion
#~ msgstr "Visualiser les plages de maintenance des services"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -15229,7 +15229,7 @@ msgid "Current state duration"
 msgstr "Durée de l'état actuel"
 
 msgid "Downtime duration"
-msgstr "Durée du temps d'arrêt"
+msgstr "Durée de la maintenance"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15232]:
```suggestion
msgstr "Durée de la plage de maintenance"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 9]
```diff
@@ -15271,13 +15271,13 @@ msgid "Last 31 days"
 msgstr "31 derniers jours"
 
 msgid "Set downtime on"
-msgstr "Programmer un temps d'arrêt sur"
+msgstr "Programmer une maintenance sur"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15274]:
```suggestion
msgstr "Programmer une plage de maintenance sur"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 10]
```diff
@@ -15271,13 +15271,13 @@ msgid "Last 31 days"
...
 
 msgid "Downtime set by"
-msgstr "Temps d'arrêt programmé par"
+msgstr "Maintenance programmée par"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15277]:
```suggestion
msgstr "Plage de maintenance programmée par"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 11]
```diff
@@ -15271,13 +15271,13 @@ msgid "Last 31 days"
...
 
 msgid "Set downtime on services attached to host"
-msgstr "Programmer le temps d'arrêt sur les services attachés à l'hôte"
+msgstr "Programmer une plage maintenance sur les services attachés à l'hôte"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15280]:
```suggestion
msgstr "Programmer une plage de maintenance sur les services attachés à l'hôte"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 12]
```diff
@@ -15364,7 +15364,7 @@ msgid "Acknowledge command sent"
 msgstr "Commande d'acquittement envoyée"
 
 msgid "Downtime command sent"
-msgstr "Commande de temps d'arrêt envoyée"
+msgstr "Demande de maintenance envoyée"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15367]:
```suggestion
msgstr "Commande de configuration de plage de maintenance envoyée"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/help.po [hunk 13]
```diff
@@ -661,7 +661,7 @@ msgid ""
 "Duration of retention regarding downtimes stored in database. 0 means that no "
 "retention will be applied."
 msgstr ""
-"Durée de rétention concernant les temps d'arrêts stockés en base de données. 0 "
+"Durée de rétention concernant les plages de maintenance stockés en base de données. 0 "
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/help.po:664]:
```suggestion
"Durée de rétention concernant les plages de maintenance stockées en base de données. 0 "
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/help.po [hunk 14]
```diff
@@ -1099,18 +1099,18 @@ msgstr ""
...
 "[Set downtimes on services attached to hosts] option is enbaled by default."
 msgstr ""
 "Active cette option par défaut dans le formulaire. Cette option permet de "
-"programmer un temps d'arrêt aux services liés à l'hôte."
+"programmer une plage de maintenance aux services liés à l'hôte."
 
 #: centreon-web/www/include/Administration/parameters/engine/help.php:73
 msgid "Default duration of scheduled downtimes."
-msgstr "Durée par défaut des temps d'arrêt planifiés."
+msgstr "Durée par défaut des plages de maintenance planifiés."
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/help.po:1113]:
```suggestion
msgstr "Durée par défaut des plages de maintenance planifiées."
```

**Comment by Thebarda** [comment]
Everything LGTM (please do not care about white balance)
test-verbs-2
test-verbs-3
test-verbs-1

I have one remark about that traduction that does not support accent. I guess this is not related to this PR
test-verbs-4

**Pull request merged**